### PR TITLE
VAOS cancel appointment integration tests

### PR DESCRIPTION
**Pull request by jbalboni**
## Description
This adds integration tests for cancelling appointments

## Acceptance criteria
- [ ] Builds passes

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/actions/appointments.js [hunk 1]
```diff
@@ -84,10 +85,15 @@ async function getAdditionalFacilityInfo(futureAppointments) {
   // Get facility ids from non-VA appts or requests
   const nonVaFacilityAppointmentIds = futureAppointments
     .filter(
-      appt => appt.vaos?.videoType || appt.vaos?.isCommunityCare || !appt.vaos,
```
Comment (jbalboni) [inline src/applications/vaos/actions/appointments.js:87]: This was keeping us from downloading facility info for video appointments, which we need for the cancel modal

File: src/applications/vaos/utils/selectors.js [hunk 2]
```diff
@@ -338,12 +336,8 @@ export function getCancelInfo(state) {
     !appointmentToCancel?.vaos?.videoType
   ) {
     // Confirmed in person VA appts
-    facility =
```
Comment (jbalboni) [inline src/applications/vaos/utils/selectors.js:341]: Another bug getting facility info for VA appointments into the cancel modal

File: src/applications/vaos/components/cancel/CancelAppointmentFailedModal.jsx [hunk 3]
```diff
@@ -31,12 +31,6 @@ export default function CancelAppointmentFailedModal({
         </p>
       )}
       <p>
-        {appointment.clinicName ? (
```
Comment (jbalboni) [inline src/applications/vaos/components/cancel/CancelAppointmentFailedModal.jsx:34]: This wasn't working and I don't think it's actually useful here anyway.

**Review by narin — APPROVED** [review]
LGTM

**Pull request merged**